content-security-policy/generic/generic-0_1-script-src.sub.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
     <title>default-src should cascade to script-src directive</title>
-    <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline'; report-uri  ../support/report.py?op=put&reportID={{$id}}">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline'; report-uri  ../support/report.py?op=put&reportID={{uuid()}}">
```

Comment:
Actually this test doesn't seem to actually check whether the report is sent or not and instead relies on the securitypolicyviolation event. I would just remove the entire report-uri directive.

Reply:
Got it